tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -271,7 +271,7 @@ def build_jobspec(self):
                     cmake_arch_option = '-DOPENSSL_NO_ASM=ON'
                 return create_docker_jobspec(
                     self.name,
-                    'tools/dockerfile/grpc_artifact_linux_%s' % self.arch,
+                    'tools/dockerfile/grpc_artifact_centos6_%s' % self.arch,
```

Comment:
For testing that this works, it would be good to also add rollback of https://github.com/grpc/grpc/pull/21481 in this PR.

Reply:
Definitely, one of reasons why I'm trying this is to remove the hack detouring Linux distribution issues. I'll try removing #21481 once this PR and abseil PR (#20184) are merged.